.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: python
+
 python:
 - 2.7
```

Comment:
Have we decided not to test for py2.7?

Reply:
I think 2.7 support is still needed as of our last conversation.